taskcluster/mozillavpn_taskgraph/transforms/release_index.py

```diff
@@ -58,13 +61,17 @@ def add_index_routes(config, tasks):
             routes = BRANCH_INDEX_ROUTES
             branch = config.params["head_ref"]
             if branch.startswith(_GIT_REFS_HEADS_PREFIX):
-                branch = branch[len(_GIT_REFS_HEADS_PREFIX):]
-            branch = branch.replace("/",".") # We can't have "/" in the index -> so releases/2.9.0 needs to be releases.2.9.0
+                branch = branch[len(_GIT_REFS_HEADS_PREFIX) :]
+            branch = branch.replace(
+                "/", "."
+            )  # We can't have "/" in the index -> so releases/2.9.0 needs to be releases.2.9.0
             context["branch"] = branch
 
-        elif config.params["tasks_for"] == "github-release":
+        elif config.params["tasks_for"] == "action" and config.params.get(
+            "shipping_phase"
```

Comment:
Are we actually making new builds as part of relpro graphs?

Reply:
Oh hmm.. You mean to say that we aren't making new builds as part of relpro, but we were making them as part of `github-release`, so now builds will be missing from this index?

Yeah.. seems like that might be the case. I suppose the signed builds should be good enough?

Reply:
Do we know if anything's using that index?  (I think any consumers should be pointed at https://archive.mozilla.org/pub/vpn/releases/, and this `RELEASE_INDEX_ROUTES` removed)

Reply:
@mcleinman any thoughts on ^ ?

Reply:
Apologies, which index is this one in question?

Reply:
This is about things like https://firefox-ci-tc.services.mozilla.com/tasks/index/mozillavpn.v2.mozilla-vpn-client.release.v2.26.0.latest

Reply:
I think that is fine. (I just checked - There are two things we use that for in the RC process, but we can get those files from the commit checks instead.)